test/document_stores/test_weaviate.py

```diff
@@ -1,8 +1,12 @@
+import uuid
+from unittest.mock import patch
```

Comment:
The linter is going to complain for the unused import :wink:

Reply:
Mine was complaining too but I just ignored it 🤦